shopfloor/services/service.py

```diff
@@ -267,7 +267,10 @@ def _get_openapi_default_parameters(self):
         # because it's a demo key.
         demo_api_key = self.env.ref(
```

Comment:
isn't this useless as we don't have the key at all?
Shall we do something better? Either add the key back for demo only or trash this bits and put "PUT_YOUR_KEY_HERE" in value?

Reply:
IMO it's more related to base_rest, you can expect the same behavior in shopinvader as well. If something is needed to be done; it should be on base_rest.

I'm in favor to remove this demo_api_key from shopfloor;  I don't use this feature btw.

Reply:
It's useful when testing the module with demo data

Reply:
It's useful when testing the module with demo data

Reply:
alright, fine then.